ert_storage/endpoints/compute/misfits.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import pandas as pd
+from uuid import UUID
+from typing import Any, Mapping, Optional, List
+import sqlalchemy as sa
+from fastapi import (
+    APIRouter,
+    Depends,
+)
+from ert_storage.database import Session, get_db
+from ert_storage import database_schema as ds, json_schema as js
+
+router = APIRouter(tags=["misfits"])
+
+
+def _calculate_misfit(obs_value, response_value, obs_std, x_value):
```

Comment:
The output of the misfit calculation is a dataframe where the indices are `x_value` and columns are `value`. I don't understand why we generate an intermediate `dict`.

I realise this is copied verbatim from new storage, but still.